truss-chains/truss_chains/definitions.py

```diff
@@ -75,6 +75,7 @@ class SafeModel(pydantic.BaseModel):
         arbitrary_types_allowed=False,
         strict=True,
         validate_assignment=True,
+        extra="allow",
```

Comment:
Now we need this instead of the default `"ignore"` behavior because `predict_url` look-ups will fail for older Truss CLI versions due to the `name`/`display_name` discrepancy. It was failing silently before but now we need to make older versions fall back to `predict_url` on `ServiceDescriptor` which is why this `"allow"` flag is necessary.